src/Css/Stylesheet.php

```diff
@@ -1612,39 +1612,26 @@ private function _parse_font_face($str)
      */
     private function _parse_properties($str)
     {
-        $properties = preg_split("/;(?=(?:[^\(]*\([^\)]*\))*(?![^\)]*\)))/", $str);
         $DEBUGCSS = $this->_dompdf->getOptions()->getDebugCss();
 
-        if ($DEBUGCSS) {
-            print '[_parse_properties';
-        }
+        if ($DEBUGCSS) print '[_parse_properties';
 
-        // Create the style
+        // Split on non-escaped semicolons which are not followed by a single
+        // closing parenthesis, to support semicolons as part of `url()`.
+        // As a consequence, semicolons and closing parentheses should be
+        // escaped if used in a string
+        $properties = preg_split("/(?<!\\\\); (?! [^\(]* (?<!\\\\)\) )/x", $str);
```

Comment:
I haven't actually tested but just a read through and I think one thing that does not parse correctly (but that wasn't parsed correctly previously either) is a semi-colon in a string, e.g. `content: "dompdf;";`.

Reply:
Yes, didn’t intend to change anything about that. But together with the updates to `Style::parse_string()` here, you can use it in escaped form now, at  least. So

```
content: "dompdf\;";
```

does work.